src/main/java/e0148811/duke/Deadline.java

```diff
@@ -0,0 +1,27 @@
+package e0148811.duke;
+
+import java.time.LocalDate;
+
+public class Deadline extends Task {
+    private LocalDate deadline;
+
+    public Deadline(String description, LocalDate deadline) {
```

Comment:
Personally, I prefer just having one constructor as your situation does not warrant multiple constructors. You can have just one constructor that takes in isdone, description and deadline and for cases where you do not need to pass in isdone's value, just put in false

Reply:
Would it not be better to have 2 constructors if most of the time you the isDone value is false upon construction? It's only not false when you're importing from file. This would allow shorter constructors most of the time which would be better.